packages/features/audit-log/saveAuditLogWorker.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  BookingCreateAuditLogger,
+  BookingUpdateAuditLogger,
+} from "@calcom/features/audit-log/bookingAuditLogger";
+import {
+  EventTypeCreateAuditLogger,
+  EventTypeUpdateAuditLogger,
+  EventTypeDeleteAuditLogger,
+} from "@calcom/features/audit-log/eventTypeAuditLogger";
+import type { BookingWithAttendees } from "@calcom/features/audit-log/types/BookingAuditLogTypes";
+import { BookingAuditLogOption } from "@calcom/features/audit-log/types/BookingAuditLogTypes";
+import { EventTypeAuditLogOption } from "@calcom/features/audit-log/types/EventTypeAuditLogTypes";
+import { prisma } from "@calcom/prisma";
+import type { EventType, Prisma } from "@calcom/prisma/client";
+
+async function saveAuditLogWorker(input: any) {
```

Comment:
I feel that we should avoid using `any` here'. This is very prone to critical errors.

Reply:
Done.